matchbox_socket/src/webrtc_socket/socket.rs

```diff
@@ -603,6 +671,29 @@ impl WebRtcSocket<SingleChannel> {
     pub fn send(&mut self, packet: Packet, peer: PeerId) {
         self.try_send(packet, peer).expect("Send failed");
     }
+
+    /// Returns whether the socket channel is closed
+    pub fn is_closed(&self) -> bool {
+        self.channel(0).is_closed()
+    }
+}
+
+impl WebRtcSocket<MultipleChannels> {
+    /// Returns whether any socket channel is closed
+    pub fn any_closed(&self) -> bool {
```

Comment:
```suggestion
    pub fn has_closed_channels(&self) -> bool {
```

Maybe this would be less ambiguous?

Reply:
Going to go with `any_closed` but if you have a strong opinion later we can revisit